### PR TITLE
add a bunch of stuff to the intro

### PR DESCRIPTION
**Pull request by Gnurdle**
Tried to put some meat into page 1, something that would be interesting for a developer to hear about the lanuage

### CODE REVIEW

File: Pages/1-intro.md [hunk 1]
```diff
@@ -18,12 +125,37 @@ java version "1.8.0_40"
...
+
+So whether via brew, package manager, shell script or .bat file from the leiningen site you are properly setup with with leiningen if you can do the following from the command line:
+
+```
+▶ lein version
+Leiningen 2.7.0 on Java 1.8.0_102 OpenJDK 64-Bit Server VM
+```
+or something similar
 
-## Installing Lein
```
Comment (rubygeek) [inline Pages/1-intro.md:25]: how about leaving the instructions for installing lein?

File: Pages/1-intro.md [hunk 2]
```diff
@@ -1,13 +1,120 @@
...
+
+## Getting Started
+Getting Clojure up and running on your computer is delightfully simple.
+
+There are two things you need to get there.
+
+   - first, you need Java, since that that is what all things Clojure are made from.  You will likey want to opt for the entire JDK (developer kit) rather than just the JRE (runtime only).
+   
+   - second, you need a program called Leiningen, which will bootstrap the rest of the process.
+
```
Comment (rubygeek) [inline Pages/1-intro.md:117]: i like all this dialog :)

File: Pages/1-intro.md (hunk 1 above)
Comment (Gnurdle) [reply Pages/1-intro.md:25]: I didn't think I had taken anything away, just fleshed it out a bit

Comment (Gnurdle) [reply Pages/1-intro.md:25]: I think paragraph starting with "Failing that..." ends up being the old procedure

**Pull request merged**